src/components/TodoForm/TodoForm.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { UserPropTypes } from '../propTypes/UserPropTypes';
+
+export class TodoForm extends React.PureComponent {
+  state = {
+    title: '',
+    userName: '',
+    titleError: false,
+    userNameError: false,
+  }
```

Comment:
create a field 'errors' in state for all errors

`errors: {
   error1, error 2
}`